rpc/src/server/mod.rs

```diff
@@ -165,6 +165,7 @@ where
 }
 
 /// BaseChannel lifts a Transport to a Channel by tracking in-flight requests.
+#[pin_project]
```

Comment:
Can the field `transport` be annotated with `#[pin]` instead of the manual fn below?

Reply:
Done.